src/libAtomVM/nifs.c

```diff
@@ -6354,6 +6420,36 @@ static term nif_erlang_list_to_bitstring_1(Context *ctx, int argc, term argv[])
     return nif_erlang_list_to_binary_1(ctx, argc, argv);
 }
 
+static term nif_erlang_alias(Context *ctx, int argc, term argv[])
+{
+    UNUSED(argc);
+    UNUSED(argv);
+
+    if (UNLIKELY(memory_ensure_free_opt(ctx, TERM_BOXED_PID_REF_SIZE, MEMORY_CAN_SHRINK) != MEMORY_GC_OK)) {
+        RAISE_ERROR(OUT_OF_MEMORY_ATOM);
+    }
+
+    uint64_t ref_ticks = globalcontext_get_ref_ticks(ctx->global);
+    term pid_ref = term_make_pid_ref(ctx->process_id, ref_ticks, &ctx->heap);
+    struct Monitor *monitor = monitor_alias_new(ref_ticks, CONTEXT_MONITOR_ALIAS_EXPLICIT_UNALIAS);
+    if (IS_NULL_PTR(monitor)) {
+        RAISE_ERROR(OUT_OF_MEMORY_ATOM);
+    }
+    context_add_monitor(ctx, monitor);
+    return pid_ref;
+}
+
+static term nif_erlang_unalias(Context *ctx, int argc, term argv[])
```

Comment:
Shouldn't be named with `_1` suffix? Typically, NIFs without aliases handle multiple arities.